Sources/StreamChatUI/ChatThreadList/ChatThreadListViewModel.swift

```diff
@@ -0,0 +1,221 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Combine
+import StreamChat
+import UIKit
+
+/// The ViewModel for the `ChatThreadListView`.
+open class ChatThreadListViewModel: ObservableObject, ChatThreadListControllerDelegate, EventsControllerDelegate {
+    /// Context provided dependencies.
+    public var chatClient: ChatClient!
```

Comment:
This is the view model exactly from SwiftUI, the only difference is that it does not have the Injected property wrapper